### DISCUSSION TITLE
localhost are pointing to strange places

### DISCUSSION THREAD

**Discussion by hhx2789** [mircokroon/minecraft-world-downloader#249] [Q&A]
I'm putting in the ip and and it won't connect
![image](https://user-images.githubusercontent.com/95308998/144107484-985890e4-a996-44bc-83ab-279c1756fc9b.png)
![image](https://user-images.githubusercontent.com/95308998/144107502-8b333a25-5f9f-4930-9676-ad3d9c60a686.png)
I have never signed up for a similar service
How do I remove
![image](https://user-images.githubusercontent.com/95308998/144107678-57ffc7e2-900e-4291-b47f-74161aa130f3.png)

**Comment by mircokroon** [comment]
Might be something on the server-side, not sure what could be causing this though as the domain name should be set correctly when connecting. The server might have a DNS record that's not handled correctly?

**Comment by WhynotFulp** [comment]
Hey did you ever figure out how to connect to local host?
